### ISSUE TITLE
Unable to load files due to redirection

### ISSUE THREAD

**Issue by laozi999** [sophron/wifiphisher#394]
phishinghttp.py makes sure that every request gets redirected to the index.html file in the selected folder, including external js files, css files, etc. This keeps me from loading external scripts. I've tried to write the path several ways `(./css/style.css)` `(http://10.0.0.1:8080/css/style.css)` none of them work.

Console dump:

`Password fields present on an insecure (http://) page. This is a security risk that allows user login credentials to be stolen.[Learn More] <unknown>`

**`The stylesheet http://10.0.0.1:8080/index was not loaded because its MIME type, "text/html", is not "text/css". 10.0.0.1:8080`**

`SyntaxError: expected expression, got '<' framework7.min.js:1:0`

`SyntaxError: expected expression, got '<' kitchen-sink.js:1:0`

`ReferenceError: $ is not defined 10.0.0.1:8080:711:1`

`downloadable font: rejected by sanitizer (font-family: "lucy" style:normal weight:normal stretch:normal src index:0) source: http://10.0.0.1:8080/fonts/segoeui.ttf 10.0.0.1:8080:12:12`

`downloadable font: rejected by sanitizer (font-family: "lucy-b" style:normal weight:normal stretch:normal src index:0) source: http://10.0.0.1:8080/fonts/segoeui-bold.ttf`

As you can see the url of the css file was replaced with 'http://10.0.0.1:8080/index' as well.

Is there a way to just redirect the user to the folder and let us handle the default index page using .htaccess like a standard webserver?

**Comment by sophron** [comment]
Hi @laozi999,

We don't want to include any external scripts (we assume that the user does not has any connection to the Internet).

You need to put your files in the `static` directory. See how the other templates are structured (e.g. https://github.com/sophron/wifiphisher/tree/master/wifiphisher/data/phishing-pages/firmware-upgrade).

**Comment by laozi999** [comment]
Hi @sophron 

Sorry, I didn't mean to say the scripts are external files, they are still local.

I was able to successfully load stylesheets via `http://10.0.0.1/css/styles.css` in previous versions of wifiphisher. I did have to change some of the code inside phishinghttp.py (which is undesirable).

Will try using the **static** folder now, thanks!

**Comment by laozi999** [comment]
And if it works, I will make a PR for the login-modal scenario immediately :+1:

**Comment by laozi999** [comment]
Works like a charm.

This is quite consistent, it'd be nice to have an empty template for users to copy and make changes to so they can build their own without going through the hassle I went through.

I have patched something quite important, namely the -qS (--quitonsuccess) command, which triggers a **sys.exit(0)**, this also wipes the captured credentials output when wifiphisher shuts down, shit!

I will submit the PR now.
